### ISSUE TITLE
Single Step Prediction Possible?

### ISSUE THREAD

**Issue by JuliaGast** [CunchaoZ/CyGNet#6]
Hello,

From the Code and from the Paper I understand, that the Cygnet model always predicts multiple steps (all steps from the test set) ahead, without feeding the ground truth information in between, is this correct?

Is it possible, to set some parameter, or easily modify the code such that Cygnet also does single step prediction?
With this I mean:

for each timestep t in test_set:
* Predict sub/ ob for all triples in t -> compute scores for these triples
* **Feed the ground truth** for t
* t = t+1

Similar to the setting described in RE-GCN, 5.14:
https://arxiv.org/pdf/2104.10353.pdf

> To conduct the multi-step inference [ 16 ] in the validation set and
> the test set, we evaluate the performance of RE-GCN with the evo-
> lutional embeddings of entities and relations at the final timestamp
> of the training set as the input of score functions following [43 ].
> Besides, we also report the results of the models **with ground truth
> history** given during multi-step inference on the test set, namely,
> w. GT. 

Best
Julia
